glomap/estimators/view_graph_calibration.cc

```diff
@@ -55,6 +55,11 @@ void ViewGraphCalibrator::Reset(
   focals_.reserve(cameras.size());
   for (const auto& [camera_id, camera] : cameras) {
     focals_[camera_id] = camera.Focal();
+    // Avoid zero focal length
+    if (focals_[camera_id] < 1e-3) {
```

Comment:
Why and when does this happen?

Reply:
this part also confuses me. I retrieved database with colmap, and one image gives me 0 focal length..